tests/bgp/test_bgp_update_timer.py

```diff
@@ -137,7 +137,7 @@ def bgp_update_packets(pcap_file):
         """Get bgp update packets from pcap file."""
         packets = sniff(
             offline=pcap_file,
-            lfilter=lambda p: bgp.BGPHeader in p and p[bgp.BGPHeader].type == 2
+            lfilter=lambda p: IP in p and bgp.BGPHeader in p and p[bgp.BGPHeader].type == 2
```

Comment:
Why are we filtering only on IP and not ipv6?

Reply:
Because in this test, we only test over ipv4 neighbors, all the updates are supposed to be in ipv4.